Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG BASE_IMAGE=ubuntu:22.04
+FROM ${BASE_IMAGE} AS build-stage
+
+ARG PJ_TAG=3.9.2
+
+###############################################################################
+# Install general dependencies.
+###############################################################################
+RUN apt update && apt install -y build-essential cmake git 
```

Comment:
&& apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*

Reply:
need to add cleany up to not carry the tmp file into the container

Reply:
That's good advice in general, thanks.
But in this particular case, I'm not using the final image. And in fact Ubuntu is used as a build stage.

Do you still think it's necessary?